.circleci/config.yml

```diff
@@ -39,9 +39,18 @@ jobs:
       - attach_workspace:
           at: ~/project
       - run:
-          name: Lint and Security
+          name: Lint
           command: yarn test:lint
 
+  audit:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: Security Audit
+          command: npm install --package-lock-only && npm audit --audit-level=moderate
```

Comment:
I wish we could have an easier way to convert `yarn.lock` into `package.lock.json`